support-frontend/assets/pages/[countryGroupId]/guardianLightLanding/components/headerCards.tsx

```diff
@@ -88,21 +92,27 @@ export function HeaderCards({ geoId }: HeaderCardsProps): JSX.Element {
 		];
 	const formattedPrice = simpleFormatAmount(currency, price);
 
-	const card1UrlParams = new URLSearchParams({
+	const guardianLightParams = {
 		product: 'GuardianLight',
 		ratePlan: contributionType,
 		contribution: price.toString(),
-	});
+	};
+	const card1UrlParams = new URLSearchParams(
+		returnLink
+			? {
+					...guardianLightParams,
+					returnAddress: returnLink,
+			  }
+			: guardianLightParams,
+	);
 	const checkoutLink = `checkout?${card1UrlParams.toString()}`;
 	const card1 = {
 		link: checkoutLink,
 		productDescription: productCatalogGuardianLight().GuardianLight,
 		ctaCopy: `Get Guardian Light for ${formattedPrice}/month`,
 	};
-
-	const returnLink = `https://www.theguardian.com/${geoId}`; // ToDo : store and use return path
 	const card2 = {
-		link: returnLink,
+		link: returnLink ?? `https://www.theguardian.com/${geoId}`,
```

Comment:
I'm not 100% sure how the geo/edition stuff works on the main website these days, but I wonder if we want to let it do its thing and not try and add `geoId` ourselves here?

Reply:
Yep it does seem to geolocate automatically without having to supply the region, this is done throughput the code so chnaged to match.